### ISSUE TITLE
[BUG] Windows Terminal shortcut not working

### ISSUE THREAD

**Issue by Helmoes** [bostrot/wsl2-distro-manager#101]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

When using the 'Open with Windows Terminal' shortcut, the selected wsl instance doesn't open in Windows Terminal. Instead there is a popup error "Windows cannot access the specified device, path or file. You may not have the appropriate permissions to access the item."

### Expected Behavior

I expect the selected wsl instance to open in a tab in Windows Terminal.

### Environment

```markdown
- Windows Version: Windows 10 Home Build 19045
- WSL Distro Manager Version: 1.6.0
- WSL Version: 1.0.3.0
- Source: GitHub releases
```

### Anything else?

<img width="592" alt="wsl2distromanager_20230220T044243" src="https://user-images.githubusercontent.com/61212162/220005208-86e1427d-a758-4caa-85fe-dde360fccdee.png">

**Comment by bostrot** [comment]
This has something to do with #23. (build 19045 is very old. can you upgrade it?)

**Comment by Helmoes** [comment]
According to [the microsoft docs](https://learn.microsoft.com/en-us/windows/release-health/release-information#windows-10-current-versions-by-servicing-option) I'm on the latest version and build: 22H2 19045.2604.

I think I mentioned the wrong build in the opening comment. 

This is the output of running winver:
![image](https://user-images.githubusercontent.com/61212162/220359344-c061d724-2a27-45ba-b80c-f549ede04b10.png)

I can't upgrade without going into the pre-release versions.

**Comment by bostrot** [comment]
Oh I see, my bad. 

Can you check whether you have Windows Terminal in your path or add it if it is not (` wt --version` or with powershell `Get-Command wt`).

**Comment by Helmoes** [comment]
My system did not recognize `wt` as a program. This did lead me to the solution though.

A week or two ago I noticed there were two python3.exe in my path. This lead to errors while using Python. This was because `%LOCALAPPDATA%\Microsoft\WindowsApps` is on the path. In that directory there are the 'aliases' that one can use to call store apps from the cli. The subdirectory `Microsoft.DesktopAppInstaller_...` has the aliases for `python.exe` and `python3.exe`. These were messing with my Python scripts. The windows settings page 'App execution aliases' allows you to toggle these aliases. Once an alias is enabled it will appear in `%LOCALAPPDATA%\Microsoft\WindowsApps`.

In the end the bug was my own fault... Not knowing what the consequences were when disabled, I disabled all aliases because 'who needs this junk'... This solved my python problem, but led to `wt` not being on the path.

Conclusion/fix: don't disable this slider.

![image](https://user-images.githubusercontent.com/61212162/220391092-23f88be7-6e74-4770-b766-d751905b3b1e.png)
